editor/project_manager.cpp

```diff
@@ -786,6 +786,7 @@ class ProjectDialog : public ConfirmationDialog {
 			_test_path();
 		}
 
+		set_size(get_minimum_size());
```

Comment:
It might be good to add a comment above the line to explain why this line is needed. Something like:

```cpp
// Reset the dialog size to its initial size. Otherwise, the dialog window would be too large
// if opening a small dialog after closing a large dialog.
```